### PR TITLE
JAVA-554

### PR DESCRIPTION
**Pull request by bsideup**

### CODE REVIEW

**Comment by tolbertam** [comment]
Can one of the admins verify this patch?

File: driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java [hunk 1]
```diff
@@ -95,6 +95,14 @@ PreparedStatement getPreparedQuery(QueryType type) {
         }
         return stmt;
     }
+    /**
+     * The {@code TableMetadata} for this mapper.
+     *
+     * @return the {@code TableMetadata} for this mapper.
```
Comment (adutra) [inline driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java]: Could you maybe add a note that this could return `null`?

**Comment by adutra** [comment]
lgtm :+1:

**Comment by bsideup** [comment]
@adutra Note added.

**Comment by adutra** [comment]
Thanks @bsideup! One last request before we merge: could you please squash your commits into a single one with a more meaningful message, e.g. "Mapper should allow access to TableMetadata (JAVA-554)." ?

**Comment by bsideup** [comment]
@adutra ok, sure. Will do it now

**Comment by bsideup** [comment]
@adutra Done:)

**Pull request merged**